mysql/init/insert_data.sql

```diff
@@ -1,19 +1,19 @@
 INSERT INTO
-    `user` (id, name, password)
+    `user` (id, name, email, password)
 VALUES
-    ("1", 'hoge@fuga.com', 'pass123');
+    ("1", '山田','hoge@fuga.com', 'pass123');
```

Comment:
```suggestion
    ("1", '山田 太郎','Taro.Yamada@example.com', 'pass123');
```
元が悪いのですが、ドメインを`@example.com`にしてもらえますか？
参考：https://qiita.com/suzutsuki0220/items/4ad83ed2e2adbb6507a4

Reply:
@iVinci-AkitoYOSHIMURA 
`@example.com`に修正しております。

Reply:
@iVinci-KenichiRan 
`'` と`"`が混在しているのが気になります。統一しましょう。
※これも元が悪い

Reply:
@iVinci-AkitoYOSHIMURA 
クォーテーションを統一しました。
80058bc